python/tk_desktop/ui/about_screen.py

```diff
@@ -71,10 +71,10 @@ def setupUi(self, AboutScreen):
         QtCore.QMetaObject.connectSlotsByName(AboutScreen)
 
     def retranslateUi(self, AboutScreen):
-        AboutScreen.setWindowTitle(QtGui.QApplication.translate("AboutScreen", "About Shotgun Desktop", None, QtGui.QApplication.UnicodeUTF8))
-        self.header.setText(QtGui.QApplication.translate("AboutScreen", "<b><big>Shotgun Desktop</big></b>", None, QtGui.QApplication.UnicodeUTF8))
+        AboutScreen.setWindowTitle(QtGui.QApplication.translate("AboutScreen", "About ShotGrid Desktop", None, QtGui.QApplication.UnicodeUTF8))
+        self.header.setText(QtGui.QApplication.translate("AboutScreen", "<b><big>ShotGrid Desktop</big></b>", None, QtGui.QApplication.UnicodeUTF8))
         self.body.setText(QtGui.QApplication.translate("AboutScreen", "Body", None, QtGui.QApplication.UnicodeUTF8))
-        self.copyright.setText(QtGui.QApplication.translate("AboutScreen", "Copyright ©2020 Shotgun Software Inc.\n"
```

Comment:
I noticed that you updated the copyright date when changed the string above -- do we want to update the copyright date here to 2021 as well?

Reply:
Yup, I missed it. Thanks.